projects/services-ogc/src/lib/owc/owc-json.service.ts

```diff
@@ -463,13 +550,59 @@ export class OwcJsonService {
         // @TODO
         break;
       case CustomLayertype:
-        // custom layers are meant to be userdefined and not easily encoded in a OWC.
+        // custom layers are meant to be user-defined and not easily encoded in a OWC.
         break;
     }
 
     return rasterLayer$;
   }
 
+  createTmsLayerFromOffering(offering: IOwsOffering, resource: IOwsResource, context: IOwsContext): Observable<CustomLayer> {
+    const layerOptions = this.getLayerOptions(offering, resource, context);
+    const tmsServerUrl = offering.operations.find(o => o.code === 'GetTiles').href;
+    const layer = new CustomLayer({
+      ... layerOptions,
+      type: 'custom',
+      custom_layer: new olVectorTileLayer({
+        declutter: true,
+        source: new olVectorTileSource({
+          attributions: this.getResourceAttribution(resource),
+          format: new olMVT(),
+          tileGrid: createXYZ({
+            minZoom: 0,
+            maxZoom: 12
+          }),
+          tilePixelRatio: 16,
```

Comment:
Is `tilePixelRatio` existing on `olVectorTileSource` https://openlayers.org/en/latest/apidoc/module-ol_source_VectorTile-VectorTile.html?  I do not find a API to set it, only a getter https://github.com/openlayers/openlayers/blob/v6.6.1/src/ol/source/VectorTile.js#L446

Reply:
Has been removed!
PS: this was just copied from our original sprint project. We should remove this attribute there, too.